### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kthiha-dev**

### CODE REVIEW

**Review by hsinhenMlion — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
-# git-geek
\ No newline at end of file
+# git-geek
+## JB
+ - Create new ReadMe file for Customer and think meaninful name for it.
```
Comment (jabezAtMlionCorp) [inline README.md:3]: comment

File: README.md [hunk 2]
```diff
@@ -1 +1,10 @@
...
+## YY
+ - Create new ReadMe file for Order and think  meaninful name for it.
```
Comment (jabezAtMlionCorp) [inline README.md:5]: comment 3

File: README.md [hunk 3]
```diff
@@ -1 +1,10 @@
-# git-geek
\ No newline at end of file
+# git-geek
+## JB
+ - Create new ReadMe file for Customer and think meaninful name for it.
+## YY
```
Comment (jabezAtMlionCorp) [inline README.md:4]: comment2

**Review by jabezAtMlionCorp — APPROVED** [review]

**Review by hsinhenMlion — APPROVED** [review]

**Review by jabezAtMlionCorp — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,10 @@
-# git-geek
\ No newline at end of file
+# git-geek
+## JB
```
Comment (yymlion) [inline README.md:2]: hi

**Review by yymlion — CHANGES_REQUESTED** [review]

**Review by hsinhenMlion — APPROVED** [review]

**Review by yymlion — APPROVED** [review]

**Pull request merged**